masonry/src/widgets/text_area.rs

```diff
@@ -796,7 +796,7 @@ impl<const EDITABLE: bool> Widget for TextArea<EDITABLE> {
     }
 
     fn accepts_focus(&self) -> bool {
-        true
+        EDITABLE
```

Comment:
Can you confirm that `accepts_focus` is *only* related to the focus chain? If so, I believe this behaviour to be correct.

The one complication is that it should ideally still be possible to select text inside a prose from only the keyboard, but I'm happy to defer that as a longer-term consideration.

Reply:
> Can you confirm that accepts_focus is only related to the focus chain?

Yup. The value is read in only two places in the code, both of them related to tab-focus (building the focus chain, and enabling `accesskit::Action::Focus`).

Reply:
Hmm... I wonder if we should allow accesskit focus, but not tab focus here